src/app/component/core/header-new/header-new.component.ts

```diff
@@ -40,10 +42,11 @@ export class HeaderNewComponent implements OnInit {
               private userService: UserService,
               private achievementService: AchievementService,
               private habitStatisticService: HabitStatisticService,
-              private languageService: LanguageService) {
-}
+              private languageService: LanguageService,
+              private clickSearch: SearchClickService) {}
 
   ngOnInit() {
+    this.clickSearch.emitter.subscribe(signal => this.isSearchClicked = signal);
```

Comment:
Please, change the subscription body.
If it is possible and does not change any functionality, write this.someFunctionName.bind(this) instead of signal => ...

Reply:
fixed